packages/blade/src/components/Carousel/Carousel.web.tsx

```diff
@@ -6,9 +6,9 @@
 import type { CSSObject } from 'styled-components';
 import styled from 'styled-components';
 import React from 'react';
-import getIn from 'lodash/get';
-import throttle from 'lodash/throttle';
-import debounce from 'lodash/debounce';
+import getIn from '~utils/lodashButBetter/get';
```

Comment:
lets rename this directory to custom-lodash-utils or something lol. I named it this for demo 🙈

Reply:
😆 this sounds good tho. fun and witty name.